server/statshouse/add_metrics_batch.h

```diff
@@ -0,0 +1,34 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2021 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+namespace statshouse {
+
+struct tl_query_payload {
+  int *buffer;
+  int len;
+};
+
+struct StatsHouseMetric {
+  int fields_mask{0};
+  std::string name;
+  //  std::vector<std::tuple<std::string, std::string>> tags; // not implemented yet
+  double counter;                // fields_mask bit #0
```

Comment:
{0.0} or just {0}
Without this, a POD type can contain trash

Reply:
fixed